.gitignore

```diff
@@ -2,3 +2,4 @@
 /.build
 /Packages
 /*.xcodeproj
+*.swp
```

Comment:
curious what brings about this local swp file?

Reply:
@jonnymacs using `vim` 😅